api4/post.go

```diff
@@ -451,7 +451,7 @@ func getPostsByIds(c *Context, w http.ResponseWriter, r *http.Request) {
 		}
 
 		post = c.App.PreparePostForClient(post, false, false)
-
+		post.StripActionIntegrations()
```

Comment:
Should `StripActionIntegrations` be called as part of `PreparePostForClient`? Are there cases where `PreparePostForClient` would _not_ be followed up by a `StripActionIntegrations`?

Reply:
This will be the best solution IMO, if it can be done. Totally avoids any further api level logic.

Reply:
I can easily make the change, but 2/5 thorough verification/tests would be out-of-scope for this PR.